MAINTAINERS.md

```diff
@@ -0,0 +1,484 @@
+    F: xlators/cluster/dht/
+
+### Erasure Coding
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    M: Xavier Hernandez  <xhernandez@redhat.com>
+    P: Ashish Pandey <aspandey@redhat.com>
+    P: Sheetal Pamecha <spamecha@redhat.com>
+    S: Maintained
+    F: xlators/cluster/ec/
+
+### Error-gen
+    S: Maintained
+    F: xlators/debug/error-gen/
+
+### FUSE Bridge
+    M: Csaba Henk <chenk@redhat.com>
+    P: Niels de Vos <ndevos@redhat.com>
+    S: Maintained
+    F: xlators/mount/
+
+### Index
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    P: Ravishankar N <ravishankar@redhat.com>
+    S: Maintained
+    F: xlators/features/index/
+
+### IO Cache
+    P: Mohammed Rafi KC <rafi.kavungal@iternity.com>
+    S: Maintained
+    F: xlators/performance/io-cache/
+
+### IO Statistics
+    S: Maintained
+    F: xlators/debug/io-stats/
+
+### IO threads
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    P: Ravishankar N <ravishankar@redhat.com>
+    S: Maintained
+    F: xlators/performance/io-threads/
+
+### Leases
+    S: Maintained
+    F: xlators/features/leases/
+
+### Locks
+    M: Xavier Hernandez  <xhernandez@redhat.com>
+    S: Maintained
+    F: xlators/features/locks/
+
+### Marker
+    M: Kotresh HR <khiremat@redhat.com>
+    S: Maintained
+    F: xlators/features/marker/
+
+### Meta
+    M: Mohammed Rafi KC <rafi.kavungal@iternity.com>
+    S: Maintained
+    F: xlators/features/meta/
+
+### Metadata-cache
+    S: Maintained
+    F: xlators/performance/md-cache/
+
+### Negative-lookup Cache
+    P: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    S: Maintained
+    F: xlators/performance/nl-cache/
+
+### gNFS
+    M: Jiffin Tony Thottan <jthottan@redhat.com>
+    P: Xie Changlong <xiechanglong@cmss.chinamobile.com>
+    P: Amar Tumballi <amar@kadalu.io>
+    S: Odd Fixes
+    F: xlators/nfs/server/
+
+### Open-behind
+    S: Maintained
+    F: xlators/performance/open-behind/
+
+### Posix:
+    S: Maintained
+    F: xlators/storage/posix/
+
+### Quick-read
+    S: Maintained
+    F: xlators/performance/quick-read/
+
+### Quota
+    S: Maintained
+    F: xlators/features/quota/
+
+### Read-ahead
+    P: Csaba Henk <chenk@redhat.com>
+    S: Maintained
+    F: xlators/performance/read-ahead/
+
+### Readdir-ahead
+    S: Maintained
+    F: xlators/performance/readdir-ahead/
+
+### Sharding
+    P: Xavier Hernandez  <xhernandez@redhat.com>
+    S: Maintained
+    F: xlators/features/shard/
+
+### Trash
+    M: Anoop C S <anoopcs@redhat.com>
+    M: Jiffin Tony Thottan <jthottan@redhat.com>
+    S: Maintained
+    F: xlators/features/trash/
+
+### Upcall
+    S: Maintained
+    F: xlators/features/upcall/
+
+### Write-behind
+    P: Csaba Henk <chenk@redhat.com>
+    S: Maintained
+    F: xlators/performance/write-behind/
+
+### Write Once Read Many
+    P: Karthik US <ksubrahm@redhat.com>
+    S: Maintained
+    F: xlators/features/read-only/
+
+### Cloudsync
+    S: Maintained
```

Comment:
Who maintains it? I don't recall anyone in the last year or two?

Reply:
Moved to 'Orphan'